plugin/trino-google-sheets/src/test/java/io/trino/plugin/google/sheets/TestGoogleSheets.java

```diff
@@ -78,4 +82,30 @@ public void testTableWithRepeatedAndMissingColumnNames()
         assertQuery("desc table_with_duplicate_and_missing_column_names", "SELECT * FROM (VALUES('a','varchar','','')," +
                 " ('column_1','varchar','',''), ('column_2','varchar','',''), ('c','varchar','',''))");
     }
+
+    @Test
+    public void testInsertIntoTable()
+            throws InterruptedException
+    {
+        long timestamp = System.currentTimeMillis();
+        UUID uuid = UUID.randomUUID();
+
+        // As we don't have DELETE support (yet), we insert into the same table every test execution and check that the INSERT can be found afterwards.
+        // This test can be improved once DELETE is supported.
+        assertUpdate("INSERT INTO insert_test VALUES (" + timestamp + ",'" + uuid + "')", 1);
```

Comment:
Please don't use static identifiers in shared services. It will lead to flaky tests when the test runs concurrently.

Reply:
We have two things that should differ every run:

* timestamp: This does not guarantee to be unique, as tests could run at the exact same time
* uuid: This additional field will guarantee, that in case the timestamps are equal, the insert will produce different tuples anyway.

I think using the tuple of both for `INSERT` and `COUNT(*)` afterwards should guarantee to not have problems during concurrently runs.

We can't create/drop or truncate any tables, so we need to insert into one global table as of now

Reply:
> We can't create/drop or truncate any tables, so we need to insert into one global table as of now

Please use `com.google.api.services.sheets.v4.Sheets` directly.

Reply:
I did a short prototype and it worked. Something along the lines
```
Sheets sheetsService = new Sheets.Builder(newTrustedTransport(), JacksonFactory.getDefaultInstance(), getCredentials()).setApplicationName("trino google sheets integration test").build();
sheetsService.spreadsheets().values().clear(TEST_METADATA_SHEET_ID, "Insert test!A2:B", new ClearValuesRequest()).execute();
```
The problem is the concurrent execution of tests. One test could delete the contents another test has written just seconds before. Having a shared services such as Google Sheets is bad here :/

The current approach never deletes any rows, so the Sheets table grows indefinitely. On the other hand it should also work indefinitely. If the test execution takes to long, the table can be truncated manually once. So my proposal for now (without `CREATE TABLE`) is to use the current solution

Reply:
Why not using random suffix to avoid concurrent failures? I disagree with the current solution.

Reply:
The current solution inserts a timestamp **and** a random uuid. Both are used when checking that the row is written:
`SELECT COUNT(*) FROM insert_test WHERE timestamp = <timestamp> AND uuid = <uuid>` is asserted to be `1`.
If i get

> Why not using random suffix to avoid concurrent failures

correctly, this can be seen as a random suffix (the uuid part)

Reply:
Sorry, had no notification for your thumbs up, totally missed it.
I implemented the mentioned mechanism, please feel free to review